react/components/EditorContainer/Sidebar/ConfigurationList/List/Card/index.tsx

```diff
@@ -18,17 +20,29 @@ const Card = ({
   intl,
   isSitewide,
   onClick,
+  onDelete,
   path,
-}: Props & ReactIntl.InjectedIntlProps) => (
-  <div
-    className={`mh5 mt5 ${!isDisabled ? 'pointer' : ''}`}
-    onClick={() => {
-      if (!isDisabled) {
-        onClick(configuration)
-      }
-    }}
-  >
-    <div className="pa5 ba br3 b--light-gray hover-bg-light-silver">
+}: Props & ReactIntl.InjectedIntlProps) => {
+  const actionMenuOptions = [
+    {
+      label: intl.formatMessage({
+        id: 'pages.editor.component-list.action-menu.delete',
```

Comment:
I believe we should use a new i18n key here, even though the value will be the same.